contacts/index.html

```diff
@@ -2,121 +2,194 @@
     <header>
-      <h1>Chat Room</h1>
+      <h1 class="align-center">Chat Room</h1>
     </header>
 
     <nav>
-      <ul>
+      <ul class="align-center">
         <li><a href="..\index.html">Home</a></li>
+        <div class="separator">|</div>
         <li>Contacts</li>
       </ul>
     </nav>
       
     <main>
-      <h2>Contacts</h2>
-      <ul>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic1.jfif" alt="pic1" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Ionescu</span>
-            <span>Alin</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic2.jfif" alt="pic2" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Sherman</span>
-            <span>Leona</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic3.jfif" alt="pic3" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Carlson</span>
-            <span>Annette</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic4.jfif" alt="pic4" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Watkins</span>
-            <span>Ervin</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div>
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic5.jfif" alt="pic5" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Pope</span>
-            <span>Philip</span>
-            <button>X</button>
+      
+      <div class="block">
+        <div class="contacts">
+          <ul class="list">
+            <h2 class="align-center">Contacts</h2>
+
+            <li>
+              <div class="person-field" class="align-center">
```

Comment:
personally, i would remove the div and add those classes to li, because i feel like you already group elements inside a container (li) and div does not add any value on that in this case